### DISCUSSION TITLE
Sort by value in a selected key?

### DISCUSSION THREAD

**Discussion by sudoforge** [TomWright/dasel#420] [Q&A] [answered]
Given the following input:

```
[
  {
    "foo": "bar",
    "value": 456
  },
  {
    "foo": "baz",
    "value": 123
  },
  {
    "foo": "quux",
    "value": 789
  }
]
```

With `jq`,I can sort by values of arbitrary keys like so:

```
➜ jq -r '. |= sort_by(.value)' /tmp/input
[
  {
    "foo": "baz",
    "value": 123
  },
  {
    "foo": "bar",
    "value": 456
  },
  {
    "foo": "quux",
    "value": 789
  }
]
```

How can I replicate this with dasel? While this example shows numeric sorting, what I'm specifically looking for is alphanumeric sorting, in case that matters.

**Comment by TomWright** [comment, marked as answer]
This isn't possible at the moment, it would make for a nice addition though
